src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,17 +1,41 @@
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as filterActions } from '../../features/filter';
+import { Status } from '../../types/Status';
 
 export const TodoFilter: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const { query, status } = useAppSelector(store => store.filter);
+
+  const handleOnStatusChange = (
+    event: React.ChangeEvent<HTMLSelectElement>,
+  ) => {
+    dispatch(filterActions.setStatus(event.target.value as Status));
+  };
+
+  const handleOnQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    dispatch(filterActions.setQuery(event.target.value));
+  };
+
+  const handleOnDelete = () => {
```

Comment:
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364